### PR TITLE
App Service Kudu - reuse affinity cookie for deployment

### PR DESCRIPTION
**Pull request by vincentdass**

### CODE REVIEW

File: Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service-kudu.ts [hunk 1]
```diff
@@ -592,21 +601,17 @@ export class Kudu {
...
-        if(!!affinityCookie) {
-           httpRequest.headers['set-cookie'] = affinityCookie; 
-        }
-
         while(true) {
             let response = await this._client.beginRequest(httpRequest);
             if(response.statusCode == 200 || response.statusCode == 202) {
                 var result = response.body;
-                tl.debug(`POLL URL RESULT: ${JSON.stringify(result)}`);
+                tl.debug(`POLL URL RESULT: ${JSON.stringify(response)}`);
```
Comment (kmkumaran) [inline Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service-kudu.ts]: why this change?

Comment (vincentdass) [reply Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service-kudu.ts]: pritning the whole response to check if the call hits the same kudu instance.
